base-theme/layouts/partials/footer.html

```diff
@@ -40,7 +40,7 @@
       <p class="font-weight-bold">
         PROUD MEMBER OF :
         <a href="https://www.oeglobal.org/" target="_blank"
-          ><img class="oeg-logo" src="/images/oeg_logo.png"
+          ><img class="oeg-logo" src="/images/oeg_logo.png" alt=""
```

Comment:
@pdpinch do you want any particular alt text for these two footer images (`mit-ol.png`, `oeg_logo.png`)?

Reply:
I thought these images are decorative (maybe or maybe not), so made alt text an empty string, but @pdpinch can confirm.

Reply:
In this case, I think we can spell out the text that's on the image, since it isn't repeated anywhere else. 
```suggestion
          ><img class="oeg-logo" src="/images/oeg_logo.png" alt="Open Education Global"
```